### ISSUE TITLE
code for alt_a and alt_q is same

### ISSUE THREAD

**Issue by alexdindikov** [skiselev/8088_bios#22]
code for alt_a and alt_q is same
here
https://github.com/skiselev/8088_bios/blob/master/scancode.inc

**Comment by alexdindikov** [comment]
https://www.fountainware.com/EXPL/bios_key_codes.htm

Alt_a $1E $00- right
Alt_q - $1E $00 wrong must be $10 $00

**Comment by skiselev** [comment]
Thank you for submitting this bug report!

I went through the entire scancode translation table, and found a couple more issues:

* On the same 'q' key, 'Ctrl-q' was returning the wrong scancode - it should have been 0x10 0x11
* Ctrl-Numpad+ should generate a 0x90 0x00. Code, and it wasn't generating any

In addition, I went through all possible combinations (no modifier/Shift/Ctrl/Alt) on an older AMI BIOS based motherboard, and compared that with 8088 BIOS, and with the table in your comment (that table has a few issues too).

Hope it is all fixed now :)
